environments/sandbox/global.tfvars

```diff
@@ -0,0 +1,39 @@
+project   = "hmcts"
+component = "frontdoor"
+location  = "uksouth"
+env       = "sandbox"
+waf_mode  = "Detection"
+
+custom_domain_name = "platform.hmcts.net"
+
+backend_domain = [
+  "uksouth.platform.hmcts.net",
+  "ukwest.platform.hmcts.net",
+]
+
+# Sample data
+frontend_with_disabled_waf_rules = [
+  {
+    name = "cmc"
+    rules = {
+      PHP  = ["933160"],
+      LFI  = ["930130"],
+      RFI  = ["931130"],
+      RCE  = ["932140", "932100", "932110", "932115", "932105", "932150"],
```

Comment:
if a rule isn't specified is it enabled?

Reply:
yes, by default all rules are enabled.